src/Authenticator/JwtAuthenticator.php

```diff
@@ -89,7 +89,7 @@ public function authenticate(ServerRequestInterface $request): ResultInterface
 
         $result = json_decode(json_encode($result), true);
 
-        $key = IdentifierInterface::CREDENTIAL_JWT_SUBJECT;
+        $key = $this->getConfig('tokenField', IdentifierInterface::CREDENTIAL_JWT_SUBJECT);
```

Comment:
Why do we have implementation specific constants on the interface? That's ugly :disappointed:.

Reply:
For 2 years now it seems: https://github.com/cakephp/authentication/commit/b24fc05f7f691205b8e84e24e9d9f0e112ad7746
But can be refactored then with cake5 or the next major in this lib.